src/routes/movies.py

```
@@ -1,10 +1,77 @@
-from fastapi import APIRouter, Depends, HTTPException, Query
-from sqlalchemy import select
+from math import ceil
+
+from fastapi import APIRouter, Depends, HTTPException, Query, status
+from sqlalchemy import select, func
 from sqlalchemy.ext.asyncio import AsyncSession
 
 from database import get_db, MovieModel
-
+from schemas import MovieDetailResponseSchema, MovieListResponseSchema
 
 router = APIRouter()
 
-# Write your code here
+
+@router.get(
+    "/movies/",
+    response_model=MovieListResponseSchema
+)
+async def get_movies(
+        page: int = Query(1, ge=1),
+        per_page: int = Query(10, ge=1, le=20),
+        db: AsyncSession = Depends(get_db),
+) -> MovieListResponseSchema | HTTPException:
```

Issue: The return type annotation should only be `MovieListResponseSchema`. Do not include `HTTPException` in the annotation, as exceptions are not returned but raised. Please update this to match the expected return type.